certbot/src/certbot/_internal/cli/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Certbot command line argument & config processing."""
 # pylint: disable=too-many-lines
+# ruff: noqa: F401
```

Comment:
this syntax ignores unused imports for the whole file. https://docs.astral.sh/ruff/rules/unused-import/ has alternate options, but imo they're unnecessary and worse.